Makefile

```diff
@@ -53,6 +53,6 @@ tag:
 .PHONY: release
 release:
 	mkdir -p releases
-	git -c tar.tar.gz.command='gzip -cn' archive --prefix=${PROGNM}-${TAG}/ -o releases/${PROGNM}-${TAG}.tar.gz ${TAG}
+	git archive --format=tar.gz --prefix=${PROGNM}-${TAG}/ -o releases/${PROGNM}-${TAG}.tar.gz ${TAG}
```

Comment:
This seems wrong, we want to keep `comand='gzip -cn'`. I don't really understand the original issue https://github.com/archlinux/archlinux-repro/issues/123 but we drop timestamps for a reason.